src/services/api.service.js

```diff
@@ -14,6 +20,17 @@ class ApiService {
       throw new Error(err);
     });
   }
+
+  async uploadFile(url, fileName) {
+    const response = (await (axios.get(url, { responseType: 'arraybuffer' }))).data;
+    this.gc.bucket(process.env.GOOGLE_BUCKET_NAME)
```

Comment:
Same here.
The reason why I say it is testing. If you use process.env variables it could be hard to test in future

Reply:
It will be done soon :)